src/components/home/Schema.js

```diff
@@ -23,7 +23,7 @@ const SchemaPart = ({ title, items }) => (
       <h4>{title}</h4>
     </div>
     <div className="schema__group">
-      {items.map(item => <SchemaItem key={item.title} {...item} />)}
```

Comment:
Why this change?

Reply:
Because of error can't map in undefined.

Reply:
Way is it undefined?

Reply:
Because of no item in current schema part i believe.